### PR TITLE
Client feed

### PR DESCRIPTION
**Pull request by AguirreCampiNicolas**

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -2,13 +2,29 @@ class HomeController < ApplicationController
...
+
+  def filter
     @products = Product.all
+
+    filter_params.each do |key, value|
+      @products = @products.where(key => true) if value == '1'
+    end
+
+    @products = @products.page(params[:page]).per(10)
+    render :index
```
Comment (JorgeLeites) [inline app/controllers/home_controller.rb]: Let's try to keep to restful routes. To filter the products we should just send the filter params to the index action

File: app/controllers/home_controller.rb [hunk 2]
```diff
@@ -2,13 +2,29 @@ class HomeController < ApplicationController
   before_action :authenticate_client!
 
   def index
+    @products = Product.page(params[:page]).per(10)
```
Comment (JorgeLeites) [inline app/controllers/home_controller.rb]: We shouldn't be accessing products in a controller called `HomeController`. Let's rename it to `ProductsController`

File: app/models/service.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class Service < ApplicationRecord
+  has_many :service_users
+  has_many :user, through: :service_users
+  has_many :products
```
Comment (JorgeLeites) [inline app/models/service.rb]: Maybe I'm not understanding the nomenclature, but in our reality we have Services, which can be of different types (Gifts, Catering, etc...). What's this relationship between products and services? What do each of them represent?

Comment (AguirreCampiNicolas) [reply app/models/service.rb]: In my mind i was thinking that if a list of products like lunchboxes appear under the gift service, then it should be related (1 service to N products).

Did I interpreted the figma wrong?

File: app/controllers/home_controller.rb (hunk 1 above)
Comment (AguirreCampiNicolas) [reply app/controllers/home_controller.rb]: I've covered this on this commit [9375356](https://github.com/AguirreCampiNicolas/onboarding/pull/1/commits/937535640e112d2f9e0a82c57177df8c0228c24c)

File: app/controllers/home_controller.rb (hunk 2 above)
Comment (AguirreCampiNicolas) [reply app/controllers/home_controller.rb]: I've covered this on this commit [9375356](https://github.com/AguirreCampiNicolas/onboarding/pull/1/commits/937535640e112d2f9e0a82c57177df8c0228c24c)

File: app/models/service.rb (hunk 3 above)
Comment (AguirreCampiNicolas) [reply app/models/service.rb]: Can you tell me if it is now resolved?

**Review by JorgeLeites — APPROVED** [review]
Looks great!

**Pull request merged**